src/main/kotlin/no/nav/familie/ef/mottak/task/ArkiverEttersendingTask.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.familie.ef.mottak.task
+
+import no.nav.familie.ef.mottak.repository.EttersendingRepository
+import no.nav.familie.ef.mottak.service.ArkiveringService
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+@TaskStepBeskrivelse(taskStepType = ArkiverEttersendingTask.TYPE, beskrivelse = "Arkiver ettersending")
+class ArkiverEttersendingTask(private val arkiveringService: ArkiveringService,
+                        private val taskRepository: TaskRepository,
+                        private val ettersendingRepository: EttersendingRepository) : AsyncTaskStep {
+
+    val logger = LoggerFactory.getLogger(this::class.java)
+
+    override fun doTask(task: Task) {
+        val journalpostId = arkiveringService.journalførEttersending(task.payload)
+        task.metadata.apply {
+            this["journalpostId"] = journalpostId
+        }
+        taskRepository.save(task)
```

Comment:
Å lagre her er egentlige litt rart, task er i utgangspunkten immutable, men det er mulig å endre metadata. 
Denne lagres etterpå i taskrameverket, men burde kanskje ha en test for det?